quantum/split_common/matrix.c

```diff
@@ -331,11 +331,8 @@ uint8_t matrix_scan(void) {
 
         matrix_scan_quantum();
     } else {
-        transport_slave(matrix + thisHand);
-#ifdef ENCODER_ENABLE
-        encoder_read();
-#endif
-        matrix_slave_scan_user();
```

Comment:
It looks like that removes the only reference to `matrix_slave_scan_user`, nowhere else calls it, so it can be removed entirely now?

Reply:
Ya, now that this feature has stabilized I'll do a cleanup pass on everything

Reply:
Added matrix_slave_scan_user back in for when mirroring is not used